src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+const min = 2;
+const max = 10;
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('tbody');
+
+appendColumn.addEventListener('click', () => {
+  [...table.rows].forEach(el => {
+    el.insertAdjacentHTML('beforeend', `
+      <td></td>
+    `);
+  });
+
+  if (table.firstChild.children.length >= max) {
```

Comment:
it's ok disable button with this way, but add `if` also before inserting element

Because user can remove `disabled` attribute with browser inspector and your button add row 11

Reply:
Check it for each listener